activerecord/test/cases/adapters/postgresql/uuid_test.rb

```diff
@@ -289,7 +289,7 @@ def test_schema_dumper_for_uuid_primary_key_with_default_nil_in_legacy_migration
       ActiveRecord::Migration.verbose = false
 
       migration = Class.new(ActiveRecord::Migration[5.0]) do
-        def version; 101 end
+        def version; 102 end
```

Comment:
Migration of version 101 is executed at https://github.com/rails/rails/pull/28046/files#diff-4b40a34b5dacebf9464c049adda25c4eL242 
And dirty migration version remains in schema_migrations table, because it simply drop the table at https://github.com/rails/rails/pull/28046/files#diff-4b40a34b5dacebf9464c049adda25c4eR252